src/gc/boehm.cr

```diff
@@ -74,6 +74,21 @@ fun __crystal_realloc(ptr : Void*, size : UInt32) : Void*
   LibGC.realloc(ptr, size)
 end
 
+# :nodoc:
+fun __crystal_malloc64(size : UInt64) : Void*
+  LibGC.malloc(size)
```

Comment:
Since malloc64 is always used for allocation, and LibC / GC malloc use SizeT. The conversion from UInt64 to SizeT happens implicitly when calling a C method? How is the failure handle when request something over SizeT on 32bits arch?

Reply:
I added a check and `raise` here. Unfortunately that raise can't be captured because it's always `call`ed and never `invoke`d in the codegen, so we should fix that later.